### PR TITLE
Node redesign timeouts

### PR DESCRIPTION
**Pull request by mjkkirschner**
following up on @QilongTang 's PR with more attempts to get this branch passing on the parallel test job.

### CODE REVIEW

**Comment by QilongTang** [comment]
Self serve on none WPF tests pass easily at https://master-15.jenkins.autodesk.com/view/DYN/job/DYN-DevCI_Self_Service/841/. So mostly some view layer or view model layer code not good.

**Comment by mjkkirschner** [comment]
Thanks Aaron

As reported in slack Friday- some test on this branch is actually crashing nunit agent when it goes to load annotationviewmodels - crash at setmaxtextwidth ()

**Comment by QilongTang** [comment]
Thanks @mjkkirschner I did some memory profiling without the group stuff and found similar results than you did, comparing to before the UI refresh, all ConnectorViewModels are still leaking which is what I was working on.

File: src/DynamoCoreWpf/Views/Core/NoteView.xaml [hunk 1]
```diff
@@ -132,7 +132,7 @@
             Visibility="{Binding Path=PinnedNode, Converter={StaticResource nullToVisibiltyConverter }, UpdateSourceTrigger=PropertyChanged}"
             >
             <Image
-                Source="/DynamoCoreWpf;component/UI/Images/Pin_NoBackground_16px.png"
+                Source="/DynamoCoreWpf;component/UI/Images/Pin_NoBackground_16_16.png"
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/Core/NoteView.xaml:135]: This change is for?

File: src/DynamoCore/Graph/Annotations/AnnotationModel.cs [hunk 2]
```diff
@@ -493,7 +493,7 @@ void SerializeCore(XmlElement element, SaveContext context)
             helper.SetAttribute("InitialTop", this.InitialTop);
             helper.SetAttribute("InitialHeight", this.InitialHeight);
             helper.SetAttribute("TextblockHeight", this.TextBlockHeight);
-            helper.SetAttribute("background", (this.Background == null ? "" : this.Background.ToString()));
+            helper.SetAttribute("backgrouund", (this.Background == null ? "" : this.Background.ToString()));
```
Comment (QilongTang) [inline src/DynamoCore/Graph/Annotations/AnnotationModel.cs:496]: Did we have this spelling mistake for XML format?

**Review by QilongTang — APPROVED** [review]
Just some questions, LGTM

File: src/DynamoCore/Graph/Annotations/AnnotationModel.cs (hunk 2 above)
Comment (mjkkirschner) [reply src/DynamoCore/Graph/Annotations/AnnotationModel.cs:496]: yes, it needs to stay, explained on slack.

File: src/DynamoCoreWpf/Views/Core/NoteView.xaml (hunk 1 above)
Comment (mjkkirschner) [reply src/DynamoCoreWpf/Views/Core/NoteView.xaml:135]: the image was renamed on this branch by someone.

**Comment by mjkkirschner** [comment]
@QilongTang unfortunately, this doesn't actually pass yet, like you said, but do you mind if I merge it anyway, so the improvements here are in for others more easily to build on?

**Comment by mjkkirschner** [comment]
well that doesn't work...
Screen Shot 2021-08-29 at 11 27 15 AM

I'm going to merge this to stop the continual failing builds 😞  - please revert if you object to that.

**Comment by QilongTang** [comment]
> @~QilongTang unfortunately, this doesn't actually pass yet, like you said, but do you mind if I merge it anyway, so the improvements here are in for others more easily to build on?~
> 
> actually I'm not 100% on the annotation Dispose calls I added, I'm pretty sure those are needed, but not sure my implementation is good yet. If you want I can move this to draft so it stops building (and failing) for now? Or merge it to get the changes in if you think thats the way to go.

hmm, did you had a graph with group saved to test? I was just using the old way of graph close and open to check memory leak like we were doing 2 years ago.

**Comment by QilongTang** [comment]
Thank you @mjkkirschner on the annotation dispose code! Let's merge and address the next set of issues.

**Pull request merged**